.github/workflows/cifuzz.yml

```diff
@@ -0,0 +1,25 @@
+name: CIFuzz
+on: [pull_request]
+jobs:
+ Fuzzing:
+   runs-on: ubuntu-latest
+   steps:
+   - name: Build Fuzzers
+     id: build
+     uses: google/oss-fuzz/infra/cifuzz/actions/build_fuzzers@master
```

Comment:
Is there a normal practice to depend on `master` for that job? The tagged version would be better, as I think (but I maybe wrong)

Reply:
This is normal practice yeah. Although now that you mention it, it does seem a little strange that this is "normal practice".

Reply:
There are some docs here https://google.github.io/oss-fuzz/getting-started/continuous-integration/